cli/azd/ci-build.ps1

```diff
@@ -120,37 +128,20 @@ if ($IsWindows) {
         $tagsFlag += ",oneauth"
     }
     Write-Host $msg
-    $buildFlags += @(
-        "-buildmode=exe",
-        # remove all file system paths from the resulting executable.
-        # Instead of absolute file system paths, the recorded file names
-        # will begin either a module path@version (when using modules),
-        # or a plain import path (when using the standard library, or GOPATH).
-        "-trimpath",
-        $tagsFlag,
-        # -extldflags=-Wl,--high-entropy-va: Pass the high-entropy VA flag to the linker to enable high entropy virtual addresses
-        ($ldFlag + "-linkmode=auto -extldflags=-Wl,--high-entropy-va")
-    )
 }
 elseif ($IsLinux) {
     Write-Host "Building for linux"
-    $buildFlags += @(
-        "-buildmode=pie",
-        ($tagsFlag + ",cfgo"),
```

Comment:
Did we eliminate the need for `cfgo`?

Reply:
Yeah, we never got around to removing it: [context](https://github.com/Azure/azure-dev/pull/2005#discussion_r1196929309)